force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,27 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
+        String sfString = 'Hello, Salesforce!';
         
-        // Get the length of the String        
+        // Get the length of the String
+        Integer sfStringLength = sfString.length();    
+        String sflength = String.valueOf(sfStringLength); 
         // Get a substring of the String of the first 5 characters
+        String sfStringSubs = sfString.substring(0,5);
         // Convert the String to lower case
+        String sfStringLowerCase = sfString.toLowerCase();
         
         // Return the list with the string data
+        List<String> stringData = new List<String>();
+        stringData.add(sflength);
+        stringData.add(sfStringSubs);
+        stringData.add(sfStringLowerCase);
+
         // In the first position add the string length
         // In the second position add the substring
         // In the third position add the string in lower case
 
-        return null; // Replace null with the variable you used to store the result
+        return stringData; // Replace null with the variable you used to store the result
```

Comment:
This is good. I would say that to be more concise it is a best to avoid declaring variables if you don't need them unless it really helps with readability. 

It is easier to add the results of the express directly to the List.

  strings.add(helloSalesforce.toLowerCase());